packages/usa-footer/src/test/template.html

```diff
@@ -73,48 +105,47 @@ <h3 class="usa-sign-up__heading">Sign up</h3>
           <div class="mobile-lg:grid-col-auto">
-            <img class="usa-footer__logo-img" src="../../dist/img/logo-img.png" alt="">
+            <img class="usa-footer__logo-img" src="../../../../dist/img/logo-img.png" alt="" />
           </div>
           <div class="mobile-lg:grid-col-auto">
-            <h3 class="usa-footer__logo-heading">Name of Agency</h3>
+            <p class="usa-footer__logo-heading">Name of Agency</p>
           </div>
         </div>
         <div class="usa-footer__contact-links mobile-lg:grid-col-6">
           <div class="usa-footer__social-links grid-row grid-gap-1">
             <div class="grid-col-auto">
-              <a class="usa-social-link usa-social-link--facebook" href="javascript:void(0);">
-              <span>Facebook</span>
+              <a class="usa-social-link" href="javascript:void(0);">
+                <img class="usa-social-link__icon" src="../../../../dist/img/usa-icons/facebook.svg" alt="Facebook" />
               </a>
             </div>
             <div class="grid-col-auto">
-              <a class="usa-social-link usa-social-link--twitter" href="javascript:void(0);">
-              <span>Twitter</span>
+              <a class="usa-social-link" href="javascript:void(0);">
+                <img class="usa-social-link__icon" src="../../../../dist/img/usa-icons/twitter.svg" alt="Twitter" />
               </a>
             </div>
             <div class="grid-col-auto">
-              <a class="usa-social-link usa-social-link--youtube" href="javascript:void(0);">
-              <span>YouTube</span>
+              <a class="usa-social-link" href="javascript:void(0);">
+                <img class="usa-social-link__icon" src="../../../../dist/img/usa-icons/youtube.svg" alt="YouTube" />
               </a>
             </div>
             <div class="grid-col-auto">
-              <a class="usa-social-link usa-social-link--instagram" href="javascript:void(0);">
-                <span>Instagram</span>
+              <a class="usa-social-link" href="javascript:void(0);">
+                <img class="usa-social-link__icon" src="../../../../dist/img/usa-icons/instagram.svg" alt="Instagram" />
               </a>
             </div>
             <div class="grid-col-auto">
-              <a class="usa-social-link usa-social-link--rss" href="javascript:void(0);">
-              <span>RSS</span>
+              <a class="usa-social-link" href="javascript:void(0);">
+                <img class="usa-social-link__icon" src="../../../../dist/img/usa-icons/rss_feed.svg" alt="RSS" />
               </a>
             </div>
           </div>
-          <h3 class="usa-footer__contact-heading">Agency Contact Center</h3>
+
+          <p class="usa-footer__contact-heading">[Agency Contact Center]</p>
           <address class="usa-footer__address">
             <div class="usa-footer__contact-info grid-row grid-gap">
+              <div class="grid-col-auto">[(800) 555-GOVT]</div>
```

Comment:
@mejiaj Was it intentional to remove the href on the phone number?

Reply:
I noticed the footer was using old markup, so I updated. It seems to be a regression in the footer component in StorybookJS implementation. I'll fix.

Reply:
Fixed in eb69bd399fa6c0d146fd4d5215e7b601630e42b2.